.github/ISSUE_TEMPLATE/new-gallery-example-request.md

```diff
@@ -0,0 +1,29 @@
+---
+name: New gallery example request
+about: This template is for requests for new examples
+title: 'New example to show how to <add words here>'
+labels: new example request
+assignees: ''
```

Comment:
```suggestion
```

Reply:
You don't need to add this if it dosen't do anything, we don't have it in the sunpy ones for instance.